src/components/Header/header.module.css

```diff
@@ -0,0 +1,11 @@
+.header{
+   display:grid;
+    grid-area:header;
+    grid-template-columns: 200px auto;
+    grid-template-areas: "brand headertext";
+    border:1px solid black;
```

Comment:
There is some black bottom border on your nav bar. Try removing it and see if it looks better

Reply:
fixed: border removed from the header